ptvsd/wrapper.py

```diff
@@ -790,6 +791,30 @@ def process_launch_arguments(self):
             redirect_output = ''
         self.pydevd_request(pydevd_comm.CMD_REDIRECT_OUTPUT, redirect_output)
 
+    def __parse_launch_option_value(self, value):
+        if value == 'True' or value == 'False':
+            return bool(value)
+        return unquote(value)
+
+    def __parse_launch_options(self, launch_options):
+        """VS options semicolon separated key=value pairs
+            WAIT_ON_ABNORMAL_EXIT=True|False
+            WAIT_ON_NORMAL_EXIT=True|False
+            REDIRECT_OUTPUT=True|False
```

Comment:
You're not handling REDIRECT_OUTPUT in launch request.

Reply:
I want to keep this about DJANGO. created a bug to handle redirect separately: https://github.com/Microsoft/ptvsd/issues/250

Reply:
Hmm, bug or missing feature 😄